docs/getting_started/write.md

```diff
@@ -57,6 +57,32 @@ Then, create a file `environment.json` with the following content :
     ]
 }
 ```
+!!! tip "An environment is already defined on your Chutney server?"
+    If you have already defined an environment on your Chutney server, you can import it by calling `synchroniseLocal` function of `EnvironmentSynchronizeService`:
```

Comment:
It has nothing to do with the doc, but we have a difference in spelling with synchroni**z**e and synchroni**s**e